docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,290 @@
+
+
+When deploying NGINX Ingress controller with Istio, you must deploy the envoy sidecar proxy into the NGINX Ingress controller pod.   
+The image below is what NGINX Ingress and Istio deployment looks like:    
+
+{{< img src="./img/nginx-envoy.png" alt="NGINX with envoy sidecar." >}}    
+
+
+By default for NGINX Ingress Controller, we populate the upstream server addresses with the endpoint IPs of the pods. NGINX Ingress Controller 1.11 release now supports the ability to configure NGINX Ingress CRDs (virtualServer/virtualServerRoute)to use the `service/cluster IP`. Using this flag,  NGINX Ingress will generate the .conf with the `service/cluster IP` of the service in the `upstreams/servers` section, instead of pod endpoint IPs of the pods.    
+
+To enable NGINX Ingress to route to the `Service IP`, we are going to use a new feature released in 1.11; [use-cluster-ip](https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#upstream).   
+
+Here is a simple example of what your `upstream` section will look like now in `virtualServer/virtualServerRoute`:
+
+```yaml
+upstreams:
+  - name: tea
+    service: tea-svc
+    port: 80
+    use-cluster-ip: true
+  - name: coffee
+    service: coffee-svc
+    port: 80
+    use-cluster-ip: true
+```
+
+Now NGINX Ingress `upstreams` will be populated with the `Service/cluster IP`. In the example above, the service/cluster IPs for `tea-svc` and `coffee-svc` will be added to the `upstream` configuration as the `server` addresses.
+
+## Setting up NGINX Plus Ingress controller for Istio.
+
+When deploying NGINX Plus Ingress Controller with Istio, you will need to modify your Depoloyment file to include the specific items needed to work with Istio. Those four specific lines are:
+
+```yaml
+traffic.sidecar.istio.io/includeInboundPorts: ""
+traffic.sidecar.istio.io/excludeInboundPorts: "80,443" 
+traffic.sidecar.istio.io/excludeOutboundIPRanges: "substitute_for_correct_subnet_range"
+sidecar.istio.io/inject: 'true'
+```
+
+Additional information on the above annotations can be found on Istios website.
+[Istio Service Mesh Annotations](https://istio.io/latest/docs/reference/config/annotations/)
+
+
+Your updated nginx-plus-ingress.yaml file will look something like this with the added annotations:
+
+```yaml
+apiVersion: apps/v1    
+kind: Deployment    
+metadata:    
+  name: nginx-ingress    
+  namespace: nginx-ingress    
+spec:    
+  replicas: 1    
+  selector:    
+    matchLabels:    
+      app: nginx-ingress    
+  template:    
+    metadata:    
+      labels:    
+        app: nginx-ingress    
+      annotations:    
+        traffic.sidecar.istio.io/includeInboundPorts: ""    
+        traffic.sidecar.istio.io/excludeInboundPorts: "80,443"    
+        traffic.sidecar.istio.io/excludeOutboundIPRanges: "10.90.0.0/16,10.45.0.0/16" 
+        sidecar.istio.io/inject: 'true'
+```
+
+## Install Istio

```

Comment:
I believe that it was explained to me that Istio needs to be installed first, prior to NIC.
If that is the case, then the walkthrough should begin with "ensure that Istio is installed first" and save to comment about NIC for when we reach needing to apply those settings in the workflow.

Reply:
Phrased differently.  The current flow is backward.